### PR TITLE
PG-473 Drop docs and disable build on GH pages

### PR DESCRIPTION
**Pull request by nastena1606**
With the documentation moved to a separate repo, remove the docs from this repository to avoid confusion where to contribute to docs. Also, removing a GH action and unpublishing the GH pages site with the docs bein built on Render.

deleted:    .github/workflows/doc-build.yml
	modified:   README.md
	deleted:    docs/COMPARISON.md
	deleted:    docs/REFERENCE.md
	deleted:    docs/USER_GUIDE.md
	deleted:    docs/_images/percona-favicon.ico
	deleted:    docs/_images/percona-logo.svg
	deleted:    docs/css/percona.css
	deleted:    docs/css/toctree.css
	deleted:    docs/index.md
	deleted:    docs/js/version-select.js
	deleted:    docs/overrides/404.html
	deleted:    docs/setup.md
	deleted:    requirements-doc.txt

### CODE REVIEW

**Comment by nastena1606** [comment]
And we need to [unpublish doc build on GitHub pages](https://docs.github.com/en/pages/getting-started-with-github-pages/unpublishing-a-github-pages-site)

**Review by ibrarahmad — COMMENTED** [review]
I think we need to delete the doc directory too.

**Comment by capri1989** [comment]
> I think we need to delete the doc directory too.

We would need to move the RELEASE_NOTES.md into the main folder for that but yeah I think we could do that instead of having a single folder "just" for this file.

**Comment by capri1989** [comment]
> And we need to [unpublish doc build on GitHub pages](https://docs.github.com/en/pages/getting-started-with-github-pages/unpublishing-a-github-pages-site)

Done

File: README.md [hunk 1]
```diff
@@ -78,9 +78,11 @@ The `pg_stat_monitor` should work on the latest version of both [Percona Distrib
...
-1. [User guide](https://github.com/percona/pg_stat_monitor/blob/master/docs/USER_GUIDE.md)
-2. [Comparing `pg_stat_monitor` and `pg_stat_statements`](https://github.com/percona/pg_stat_monitor/blob/master/docs/COMPARISON.md)
-3. [pg_stat_monitor view reference](https://github.com/percona/pg_stat_monitor/blob/master/docs/REFERENCE.md)
+The following are useful links in [`pg_stat_monitor` documentation](https://docs.percona.com/pg-stat-monitor/index.html):
+
+1. [User guide](https://docs.percona.com/pg-stat-monitor/user_guide.html)
+2. [Comparing `pg_stat_monitor` and `pg_stat_statements`](https://docs.percona.com/pg-stat-monitor/comparison.html)
+3. [pg_stat_monitor view reference](https://docs.percona.com/pg-stat-monitor/reference.html)
 4. [Release notes](https://github.com/percona/pg_stat_monitor/blob/master/docs/RELEASE_NOTES.md)
 5. [Contributing guide](https://github.com/percona/pg_stat_monitor/blob/master/CONTRIBUTING.md)
```
Comment (capri1989) [inline README.md:87]: Don't we have to update the contribution guide as well?

[the remark this answers is not in the document]
Comment (nastena1606) [reply README.md:87]: Done

**Comment by nastena1606** [comment]
> > I think we need to delete the doc directory too.
> 
> We would need to move the RELEASE_NOTES.md into the main folder for that but yeah I think we could do that instead of having a single folder "just" for this file.

Done

**Review by capri1989 — APPROVED** [review]
LGTM

**Pull request merged**